net/http/inc/THttpWSHandler.h

```diff
@@ -52,6 +52,9 @@ friend class THttpServer;
    /// Method called when multi-threaded send operation is completed
    virtual void CompleteWSSend(UInt_t) {}
 
+   /// Method used to accept or reject root_batch_holder.js request
+   virtual Bool_t ProcessBatchHolder(std::shared_ptr<THttpCallArg>) { return kFALSE; }
```

Comment:
Please use `override` in new classes!

Reply:
This is definition of virtual method in base class - no any override